src/utils/src/lib.rs

```diff
@@ -0,0 +1,4 @@
+// Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 OR BSD-3-Clause
+
+pub mod resource_download;
```

Comment:
Should this be `pub(crate)`?

Reply:
If this would be pub in crate, then we would not be able to use it outside of `utils`. The `resource_download` is required in external crates like `vmm`.

Reply:
Got it! Thanks, The crate here is `utils` I confused it to be `vmm`.